src/values/AbstractObjectValue.js

```diff
@@ -312,6 +312,8 @@ export default class AbstractObjectValue extends AbstractValue {
   // ECMA262 9.1.8
   $Get(P: PropertyKeyValue, Receiver: Value): Value {
     if (P instanceof StringValue) P = P.value;
+    let isComputed = (typeof P === "string" && !isNaN(P)) || P instanceof AbstractValue;
```

Comment:
Can we put this inline into the generator function since it is related to serialization? It will only be called once anyway so it shouldn't matter. However, most of the time that path will never taken so unnecessary to eagerly test for this.

However, this test is also not sufficient because it doesn't test for all possible invalid identifiers. It should test for `isValidIdentifier`. Like this: https://github.com/facebook/prepack/blob/9081d3bf3f70585411d1c9c0bc2e5db07bae532e/src/evaluators/CallExpression.js#L125

The last check here `P instanceof AbstractValue` is never going to be true because there is an invariant below that checks that `typeof P` is a string so not sure why that is needed.